paper.bib

```diff
@@ -0,0 +1,30 @@
+@book{martin2008,
+author = {Martin, Robert C.},
+title = {Clean Code: A Handbook of Agile Software Craftsmanship},
+year = {2008},
+isbn = {0132350882},
+publisher = {Prentice Hall PTR},
+address = {USA},
+edition = {1}
+}
+
+@article{perkel2022,
+author = {Perkel, Jeffrey M.},
+title = {How to fix your scientific coding errors},
+year = {2022},
+month = {02},
+pages = {172-173},
+title = {How to fix your scientific coding errors},
+volume = {602},
+journal = {Nature},
+doi = {10.1038/d41586-022-00217-0}
+}
+
+@misc{copier,
+  author = {Juan-Pablo Scaletti and Ben Felder and Jairo Llopis and Timothée Mazzucotelli and Sigurd Spieckermann and Copier Contributors},
```

Comment:
I'm still not sure what to do with the author list since I extracted it from the github page and guessed at an order. I wonder if the markdown compiler will complain if we leave author out. Then it would just point to the github repository.

Alternatively we could just do the footnote link for this one.

Reply:
I created a new discussion Q&A topic in the copier repo asking if the author list that we have is correct or not. You can see it here: https://github.com/orgs/copier-org/discussions/1542

I think that this is the best option we have to getting a final answer on this question. Once I get a response, I can update this.

Reply:
The maintainers of Copier responded and said that the author list looks correct. The only request they had was that we send them a link to the paper when it's published.